### PR TITLE
migration of rm-case-service for census-rm-case-service

### PR DESCRIPTION
**Pull request by LukeLoze1**
Why?
New fork for census-rm

What's changed
removed manifest and jenkins, renamed in pom, dockerfile and .travis

how to test:
build image locally and check still works

links:
https://trello.com/c/6arZY5kG/603-fork-rm-case-service

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
+    cp .maven.settings.xml ~/.m2/settings.xml  # This only needs to be done once to set up mavens settings file
+    mvn clean install
+    mvn spring-boot:run
+    ```
+
+# Code Styler
+To use the code styler please goto this url (https://github.com/google/google-java-format) and follow the Intellij instructions or Eclipse depending on what you use
+
+## API
+See [API.md](https://github.com/ONSdigital/rm-case-service/blob/master/API.md) for API documentation.
```
Comment (AdamHawtin) [inline README.md]: Still linking to old repo

File: README.md [hunk 2]
```diff
@@ -1 +1,33 @@
-# census-rm-case-service
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/0c2913652fd94878b4c61838b54db11e)](https://www.codacy.com/app/zekizeki/rm-case-service?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=ONSdigital/rm-case-service&amp;utm_campaign=Badge_Grade) [![Docker Pulls](https://img.shields.io/docker/pulls/sdcplatform/casesvc.svg)]()
+[![Build Status](https://travis-ci.org/ONSdigital/rm-case-service.svg?branch=master)](https://travis-ci.org/ONSdigital/rm-case-service)
+[![codecov](https://codecov.io/gh/ONSdigital/rm-case-service/branch/master/graph/badge.svg)](https://codecov.io/gh/ONSdigital/rm-case-service)
```
Comment (AdamHawtin) [inline README.md]: Badges are all pointing at the old repo still

**Review by AdeHarrison — COMMENTED** [review]
added travis env vars and temp commit to force build but travis fails with "the command "cp .maven.settings.xml $HOME/.m2/settings.xml" failed and exited with 1 during ."

**Review by davidmort — CHANGES_REQUESTED** [review]
could we get a useful PR title for this please :)

File: diagrams/case-state.puml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+@startuml
+skinparam state {
+    BackgroundColor HoneyDew
+}
+
+[*] --> SAMPLED_INIT
+[*] --> REPLACEMENT_INIT
+SAMPLED_INIT --> ACTIONABLE : activated [[https://github.com/ONSdigital/rm-case-service/blob/master/src/main/java/uk/gov/ons/ctp/response/casesvc/scheduled/distribution/CaseDistributor.java#L197 *]]
+REPLACEMENT_INIT --> ACTIONABLE : replaced [[https://github.com/ONSdigital/rm-case-service/blob/master/src/main/java/uk/gov/ons/ctp/response/casesvc/scheduled/distribution/CaseDistributor.java#L201 *]]
```
Comment (AdeHarrison) [inline diagrams/case-state.puml]: diagram points to rm-case-service

File: diagrams/case-group-state.puml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+[*] --> NOTSTARTED
+NOTSTARTED --> INPROGRESS : collection_instrument_downloaded [[https://github.com/ONSdigital/ras-frontstage-api/blob/master/frontstage_api/controllers/collection_instrument_controller.py#L33 *]]
+NOTSTARTED --> INPROGRESS : eq_launch [[https://github.com/ONSdigital/ras-frontstage-api/blob/master/frontstage_api/resources/surveys/generate_eq_url.py#L51 *]]
+NOTSTARTED --> COMPLETE : successful_response_upload [[https://github.com/ONSdigital/ras-frontstage-api/blob/master/frontstage_api/controllers/collection_instrument_controller.py#L54 *]]
+
+NOTSTARTED --> COMPLETEDBYPHONE : completed_by_phone
+NOTSTARTED --> NOLONGERREQUIRED : no_longer_required
+    
+INPROGRESS --> COMPLETE : offline_response_processed [[https://github.com/ONSdigital/rm-case-service/blob/master/src/main/java/uk/gov/ons/ctp/response/casesvc/message/impl/CaseReceiptReceiverImpl.java#L60 *]]
```
Comment (AdeHarrison) [inline diagrams/case-group-state.puml]: diagram points to rm-case-service

File: src/test/resources/curlTests.txt [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+########################################################################
+curl http://localhost:8171/cases/9a5f2be5-f944-41f9-982c-3517cfcfef3c/ -v -X GET
+404 "error":{"code":"RESOURCE_NOT_FOUND","timestamp":"20170801145321789","message":"Case not found for case id 9a5f2be5-f944-41f9-982c-3517cfcfef3c"}}
+
+
+########################################################################
+## To test the info endpoint
+########################################################################
+curl http://localhost:8171/info/ -v -X GET
+200 {"name":"casesvc","version":"10.43.0-SNAPSHOT","origin":"git@github.com:ONSdigital/rm-case-service.git","commit":"bad3532bd06ecffe0626a1a5eae8401edde719da","branch":"master","built":"2017-07-11T15:41:46Z"}
```
Comment (AdeHarrison) [inline src/test/resources/curlTests.txt]: rm-case-service

File: src/test/resources/curlTests.txt [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+
+curl http://casesvc.apps.devtest.onsclofo.uk/info -v -X GET
+200 {"name":"casesvc","version":"10.43.0-SNAPSHOT","origin":"git@github.com:ONSdigital/rm-case-service.git","commit":"bad3532bd06ecffe0626a1a5eae8401edde719da","branch":"master","built":"2017-07-11T15:41:46Z"}
```
Comment (AdeHarrison) [inline src/test/resources/curlTests.txt]: rm-case-service

File: src/test/resources/curlTests.txt [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+
+curl https://casesvc-int.apps.devtest.onsclofo.uk/info -v -k -X GET
+200 {"name":"casesvc","version":"10.43.0-SNAPSHOT","origin":"git@github.com:ONSdigital/rm-case-service.git","commit":"c155bad05e75c72575a35445cc93e1dda0677338","branch":"UNKNOWN","built":"2017-07-11T14:53:37Z"}
```
Comment (AdeHarrison) [inline src/test/resources/curlTests.txt]: rm-case-service

File: README.md (hunk 2 above)
Comment (LukeLoze1) [reply README.md]: fixed

**Review by AdeHarrison — APPROVED** [review]
LGTM

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Review by davidmort — APPROVED** [review]

**Pull request merged**